### PR TITLE
Pass container first label to koji extra metadata

### PR DESCRIPTION
**Pull request by athos-ribeiro**
We want to allow container first metadata in the container.yaml file to
be forwarded to koji. We do this by copying the whole 'go' key content
to the extra.image.go koji build metadata.

### CODE REVIEW

File: atomic_reactor/schemas/container.json [hunk 1]
```diff
@@ -67,6 +67,15 @@
...
       "default": 1
+    },
+    "go": {
+      "type": ["object", "null"],
+      "properties": {
+        "module": {
+          "type": "string"
+        }
+      },
+      "additionalProperties": true
```
Comment (twaugh) [inline atomic_reactor/schemas/container.json]: Please don't include this part... I will update #1159 instead. But good catch about additionalProperties, I had forgotten to put it in.

Comment (athos-ribeiro) [reply atomic_reactor/schemas/container.json]: Oh, I had forgotten the other commit would deal with the schema. I will remove the changes in this file and wait for #1159 to be merged.

Comment (athos-ribeiro) [reply atomic_reactor/schemas/container.json]: Done

**Comment by athos-ribeiro** [comment]
Depends on #1159

File: docs/koji.md [hunk 2]
```diff
@@ -57,6 +57,8 @@ Data which is placed here includes:
 - `build.extra.image.odcs.signing_intent_overridden` (boolean): Whether or not the signing intent used is different than the one defined in container.yaml
 - `build.extra.submitter` (string): username that submitted the build via content generator API
 - `build.owner` (string or null): username that started the task
+- `build.extra.image.go` (map): information about container first Go modules
+- `build.extra.image.go.module` (str): module name for the top-level Go language package, as in `example.com/go/packagename`
```
[the remark this answers is not in the document]
Comment (athos-ribeiro) [reply docs/koji.md]: Oh, I let that one pass. Fixed.

**Comment by athos-ribeiro** [comment]
Release notes added:
- Forward Go package information in container.yaml to koji (towards container first builds)

**Pull request merged**